source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -0,0 +1,437 @@
+
+#. Download the file :file:`clear-<release number>-live-desktop.iso`
+
+   .. note::
+
+      <release-number> is the latest |CL| auto-numbered release.
+
+#. Follow your OS instructions to create a bootable USB drive.
+
+   * :ref:`bootable-usb`
+
+.. _install-on-target-start:
+
+Install from live image
+***********************
+
+After you download and burn the live desktop image on a USB drive, follow
+these steps.
+
+#. Insert the USB drive into an available USB slot.
+
+#. Power on the system.
+
+#. Open the system BIOS setup menu by pressing the :kbd:`F2` key.
+   Your BIOS setup menu entry point may vary.
+
+#. In the setup menu, enable the UEFI boot and set the USB drive as the
+   first option in the device boot order.
+
+#. Save these settings, e.g. :kbd:`F10`, and exit.
+
+#. Reboot the target system.
+
+#. Select :guilabel:`Clear Linux OS` in the boot menu, shown in Figure 1.
+
+   .. figure:: figures/bare-metal-install-desktop-01.png
+      :scale: 50 %
+      :alt: Clear Linux OS in boot menu
+
+      Figure 1: Clear Linux OS in boot menu
+
+.. _install-on-target-end:
+
+Launch the |CL| installer
+=========================
+
+#. After the live desktop image boots, find the |CL| icon in
+   the :guilabel:`Activities` menu at left, shown in Figure 2.
+
+#. Click the icon, :guilabel:`Install Clear Linux OS`.
+
+   .. figure:: figures/bare-metal-install-desktop-02.png
+      :scale: 50 %
+      :alt: Install Clear Linux OS icon
+
+      Figure 2: Install Clear Linux OS icon
+
+#. The installer is launched, as shown in Figure 3.
+
+   .. figure:: figures/bare-metal-install-desktop-03.png
+      :scale: 50 %
+      :alt: |CL| Desktop Installer
+
+      Figure 3: |CL| OS Desktop Installer
+
+#. In :guilabel:`Select Language`, select a language from the options, or
+   type your preferred language in the search bar.
+
+#. Select :guilabel:`Next`.
+
+.. _incl-bare-metal-beta-start:
+
+Minimum installation requirements
+*********************************
+
+To fulfill minimum installation requirements, complete the
+`Required options`_. We also recommend completing `Advanced options`_.
+
+.. note::
+
+   * The :kbd:`Install` button is only highlighted **after** you complete
+     `Required options`_.
+
+   * Checkmarks indicate a selection has been made.
+
+   * An Internet connection is required. You may want to launch a browser
+     prior to installation to verify your Internet connection.
+
+|CL| Desktop Installer
+**********************
+
+The |CL| Desktop Installer Main Menu appears as shown in Figure 4. To meet the minimum requirements, enter values in all submenus for the
+:guilabel:`Required options`. After you complete them, your selections appear
+below submenus and a checkmark appears at right.
+
+.. figure:: figures/bare-metal-install-desktop-04.png
+   :scale: 100%
+   :alt: Clear Linux OS Desktop Installer - Main Menu
+
+   Figure 4: Clear Linux OS Desktop Installer - Main Menu
+
+Navigation
+**********
+
```

Comment:
Is it worth mentioning the use of the mouse here?

Reply:
Done